src/AddDiveWish.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import Button from './Button.js';
+
+export default function AddListItem({ onAddDiveWish }) {
+  return (
+    <Form
+      aria-label="Add a dive destination to your wishlist"
+      onSubmit={handleAddDiveWish}
+    >
+      <Container>
+        <Label htmlFor="destination">DESTINATION </Label>
+        <Input id="destination" name="destination" required />
+      </Container>
+      <Container>
+        <Label htmlFor="notes">NOTES</Label>
+        <Textarea id="notes" name="notes" required />
+      </Container>
+      <Button type="submit">Add to list</Button>
+    </Form>
+  );
+
+  function handleAddDiveWish(event) {
+    event.preventDefault();
+    const form = event.target;
+    const input = form.elements.destination;
+    const textarea = form.elements.notes;
```

Comment:
Maybe keep the naming that you gave the input fields in the first place. If you add any more inputs or textareas, you'd need to change the naming again.
```suggestion
    const { destination, notes }  = form.elements;
```

Reply:
Very good idea, thank you.